File: hackertracker/src/main/java/com/shortstack/hackertracker/Adapter/SpinnerAdapter.java
package com.shortstack.hackertracker.Adapter;

import android.content.Context;
import android.widget.ArrayAdapter;

/**
 * Created by Whitney Champion on 7/7/16.
 */

public class SpinnerAdapter extends ArrayAdapter<String> {

    public SpinnerAdapter(Context context, int textViewResourceId) {

        super(context, textViewResourceId);

    }

    @Override
    public int getCount() {

        int count = super.getCount();

        return count>0 ? count-1 : count ;

    }

}

File: hackertracker/src/main/java/com/shortstack/hackertracker/Adapter/SpinnerAdapter.kt
package com.shortstack.hackertracker.Adapter

import android.content.Context
import android.widget.ArrayAdapter

class SpinnerAdapter(context : Context, textViewResourceId : Int) : ArrayAdapter<String>(context, textViewResourceId) {

    override fun getCount() : Int {
        val count = super.getCount()
        return if (count > 0) count - 1 else count
    }
}